geopandas/io/tests/test_file.py

```diff
@@ -440,6 +440,7 @@ def test_to_file_schema(tmpdir, df_nybb, engine):
         assert result_schema == schema
 
 
+@pytest.mark.skipif(not HAS_PYPROJ, reason="pyproj not installed")
```

Comment:
I don't know if it is worth it, but we could define this as something like `skip_if_no_pyproj` to make this a bit shorter and less repetitive when used a lot

Reply:
I tent to slightly prefer current model. We would have to define it somewhere and always figure out where it should be imported from as some tests are in `tests`, other in `io/tests`... It would result in more code than this.